monitoring/local_monitoring/local_monitoring_demo_namespace.sh

```diff
@@ -0,0 +1,403 @@
+#!/bin/bash
+#
+# Copyright (c) 2020, 2022 Red Hat, IBM Corporation and others.
```

Comment:
@shekhar316 - Can you add a cleanup or terminate routine to undo the setup say with -t option and update the readme.

Reply:
Update the copyright year

Reply:
Updated the copyright year. 
Now, deleting the all repos and namespace quota applied with -t option.